workspace/jit-image-manipulation/trusted-sites

```diff
@@ -1,34 +0,0 @@
-i.ytimg.com/*
```

Comment:
You clearly are deleting something you do not understand ;)

Please keep this.

Reply:
This is a result of all the shuffling around this PR as been through, not my intention.